ci/infra/azure/README.md

```diff
@@ -0,0 +1,160 @@
+
+Worker nodes are never exposed to the public internet. On the opposite
+each master nodes has a public IP address by default. This allows users to
+connect to them via ssh from their computers.
+
+It's also possible disable this behaviour and make **all** the nodes private.
+This can be done setting the `create_bastionhost` variable to `true`.
+
+When this variable is set all the master nodes cease to have a public IP address.
+An [Azure Bastion](https://docs.microsoft.com/en-us/azure/bastion/bastion-overview)
+instance is created which becomes the only way to ssh into the cluster.
+
+Terraform creates also an internal DNS zone with the domain specified via the
+`dnsdomain` variable. This allows all the nodes to reach each other using
+their FQDN.
+
+### Security groups
+
+Terraform automatically creates security groups for the master and worker nodes
+that are going to allow connections only to the allowed services. The security
+rules are a 1:1 mapping of what we describe inside of SUSE CaaS Platform
+documentation.
+
+## Load balancer
+
+Terraform automatically creates a load balancer with a public IP that exposes
+the following services running on the control plane nodes:
+
+  * kubernetes API server: port 6443
+  * dex: port 32000
+  * gangway: port 32001
+
+This is exactly the same behaviour used by other deployment platforms.
+
+## Accessing the nodes
+
+A default `sles` user is created on each node of the cluster. The user has
+administrator capabilities by using the `sudo` utility.
+
+By default password based authentication is disabled. It's possible to log
+using the ssh key specified via the `admin_ssh_key` variable.
+It's also possible to enable password based authentication by specifying a
+value for the `admin_password` variable. Note well: Azure has some security
+checks in place to avoid the usage of weak passwords.
+
+When the bastion host creation is disabled the access to the master nodes of
+the cluster is just a matter of doing a ssh against their public IP address.
+
+Accessing a cluster through an Azure Bastion requires a different procedure.
+
+### Using Azure Bastion
+
+Azure bastion supports only RSA keys with PEM format. These can be created by
+doing:
+
+```
+ssh-keygen -f azure -t rsa -b 4096 -m pem
+```
+
+This will create a public and private key named `azure`. The **private** key has
+to be provided later to Azure, hence it's strongly recommended to create
+a dedicate key pair.
+
+Once the whole infrastructure is created you can connect into any node of the
+cluster by doing the following steps:
+
+  * Log into Azure portal
+  * Choose one of the nodes of the cluster
+  * Click "connect" and select "bastion" as option
+  * Enter all the required fields
+
+Once this is done a new browser tab will be open with a shell session running
+inside of the desired node.
+
+It's recommended to use Chrome or Chromium during this process.
+
+You can ssh into the first master node of the cluster of drive the creation of
```

Comment:
This line looks weird to me.
`You can ssh into the first master node of the cluster of drive the creation of
a SUSE cluster from there. Once this is done you can download a kubeconfig
file and operate the cluster without having to go through the bastion host.`

Suggest:
`You can ssh into the first bootstrapped master node to download the kubeconfig
file to operate the cluster without having to go through the bastion host.`

Reply:
changed